kubeflow/assets/dashboards/overview.json

```diff
@@ -0,0 +1,1410 @@
+{
+    "author_name": "Datadog",
+    "description": "This dashboard provides insights into your Kubeflow.\nYou can use this dashbaord to check the Kubeflow componenets stats.\n",
```

Comment:
This should match what you have in the description widget. Also my guess is that you'll need to rephrase it following [the guidelines](https://datadoghq.dev/integrations-core/guidelines/dashboards/).